docs/sphinx/releases/whats_new_1_5_0.rst

```diff
@@ -8,17 +9,542 @@
 .. _hpx_1_5_0:
 
 ===========================
-|hpx| V1.5.0 (TBD)
+|hpx| V1.5.0 (?)
 ===========================
 
 General changes
 ===============
 
+* We have added the C++20 features ``hpx::jthread`` and ``hpx::stop_token``. We
+  have also added ``hpx::stable_sort`` based on Francisco Tapia's
+  implementation.
+* We have adapted existing sychronization primitives to C++20: ``barrier``,
+  ``latch``, ``counting_semaphore``.
+* We have started using customization point objects to make algorithm extension
+  easier for the user. ``reduce``, ``copy``, ``all_of``/``any_of``/``none_of``,
+  ``fill``, ``count``, ``destroy``, ``move`` are using those CPOs.  We also have
+  adapted their corresponding ``hpx::ranges`` versions in this release,
+  deprecating the ``hpx::parallel`` ones.
+* In the perspective of continuing our work on making the hpx algorithms support
+  the ranges, we have adapted ``hpx::traits::is_range`` and others to use
+  ``hpx::traits::is_sentinel_for`` concept. We have adapted ``hpx::copy``
+  and ``hpx::stable_sort`` algorithms.
+* We have adapted ``co_await`` to C++20, it now supports ``shared_future`` and
+  have added allocator support for futures returned by ``co_return``. It is no
+  longer in the ``experimental`` namespace.
+* We added serialization support for ``std::variant`` and ``std::tuple``.
+* We continued with the modularization, it is now easier to enable/disable the
+  distributed runtime option. An important amount of headers have been
+  deprecated. We adapted the namespaces and headers we could to be closer to the
+  standard ones (:ref:`api <_public_api>`).
+* We have set the HPX C++ standard automatically to C++14 and if it needs to be
```

Comment:
This is a breaking change as HPX now requires C++14 as the minimal compilation mode.

Reply:
Yes I initially separated the change to the mimimal supported standard in the breaking change section and the new cmake option related to it in the general changes section but it makes more sense to have it as the same place, thanks :)